modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -810,7 +830,7 @@ public void testAddSystemObjectDefinition() throws Exception {
 			ObjectDefinitionScopeException.class, "Scope is null",
 			() ->
 				ObjectDefinitionTestUtil.addUnmodifiableSystemObjectDefinition(
-					TestPropsValues.getUserId(), "Test", null,
+					"Test", TestPropsValues.getUserId(), "Test", null,
```

Comment:
Hey @marcelabc, some integration tests are failing because we are always defining the ERC as "Test", but we cannot have more than one objectDefinition in a company with the same ERC.

```
com.liferay.object.service.test.ObjectFieldLocalServiceTest > testDeleteObjectField FAILED
     [exec]     com.liferay.object.exception.DuplicateObjectDefinitionExternalReferenceCodeException: Duplicate object definition with external reference code Test and company 20096
``` 

Can we talk to solve this?